index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+  </head>
+
+  <body>
+  <header>  
+    <nav class="header" id="logo">
+      <h3 class="poppins"><a href="#">My Logo</a></h3>
+      <img src="./assets/hamburger menu.png">
+    </nav>
+  </header>
+  
+  <section class ="first-wrapper">
+
+    <h1 class="poppins">I'm Victoria Glad to see you!</h1>
+
+    <p class="poppins-regular">
+      I'm a software developer! I can help you build a product, feature 
+      or website. Look through some of my work and experience! If you like 
+      what you see and have a project you need coded, do not hesitate to contact me.
+    </p>
+    <h3 class="poppins-regular">LET'S CONNECT</h3>
+      <ul class="social-media-icons">
+        <li class="a"><img  src="./assets/twitter.svg"></li>
```

Comment:
[Optional] You could nest your social media icons in anchor tags so that you can link them to social media links later on.

Reply:
> [Optional] You could nest your social media icons in anchor tags so that you can link them to social media links later on.

**[OPTIONAL]:** To make your PRs descriptions and review comments aesthetically look professional and clean :sparkles: I suggest implementing the GitHub markdown. Here's a link with [useful content/tips to assist you in this matter](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)  :bulb: :smile: